### PR TITLE
README: add badge for appveyor (Windows CI) status

### PR DESCRIPTION
**Pull request by rosenhouse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.org/containernetworking/cni.svg?branch=master)](https://travis-ci.org/containernetworking/cni)
+[![Linux Build Status](https://travis-ci.org/containernetworking/cni.svg?branch=master)](https://travis-ci.org/containernetworking/cni)
+[![Windows Build Status](https://ci.appveyor.com/api/projects/status/e6jkofdn81d3ey1q/branch/master?svg=true)](https://ci.appveyor.com/project/rosenhouse/cni-caa6b)
```
Comment (bboreham) [inline README.md]: is this linked to a personal ('rosenhouse') account?

Comment (rosenhouse) [reply README.md]: yeah....it's a personal appveyor account, but linked to the real upstream github source repo.

appveyor doesn't appear to support github orgs being owners of appveyor projects.  one option is we can create a "cni-ci-bot" account on github, and sign that up for an appveyor account.  then us maintainers need to share ownership of the credentials for that github account.

thoughts?

**Comment by rosenhouse** [comment]
Updated integration to use a shiny new [**bot account**](https://github.com/cni-bot).
Bot creds are in the CNCF's LastPass account, in a folder called `Shared-CNI`

**Comment by rosenhouse** [comment]
@bboreham @squeed ok to merge this one too?

**Pull request merged**